txlib/api/tests/test_base_model.py

```diff
@@ -15,6 +16,10 @@ class DummyModel(BaseModel):
 class TestBaseModel():
     """Test the base model for the Transifex model wrappers."""
 
+    @pytest.fixture(autouse=True)
+    def auto_clean_registry(self, clean_registry):
```

Comment:
Is this needed/used at all?

Reply:
This run for every test (because of `autouse=True`).

Reply:
It does nothing though. I tried removing it and tests still pass. Is this a placeholder for something?